lambda_functions/job_monitor/job_monitor.py

```diff
@@ -13,6 +13,8 @@
 GET_USERJOBS_CMD = 'qstat -wu {}'
 GET_JOBINFOEXTRA_CMD = 'qstat -f {}'
 
+# pylint: disable=W0107
```

Comment:
In future when dealing with pylint errors:
- Fix the error instead of adding a disable (In this case, it's already been fixed by removing `pass` from line 21)
- If it does make sense to disable pylint for an error, use the name instead of the error code, ie: `pylint: disable=unnecessary-pass`

For now it's minor extra noise and can be cleaned up in a future PR.